MapboxCoreNavigation.podspec

```diff
@@ -39,10 +39,20 @@ Pod::Spec.new do |s|
 
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
-  s.swift_version = "4.0"
 
   s.dependency "MapboxDirections.swift", "~> 0.17"
   s.dependency "MapboxMobileEvents", "~> 0.3"
   s.dependency "Turf", "~> 0.0.4"
 
+  # The CocoaPods podspec spec is not backwards compatible, so, when they add new parameters
+  # (like `swift_version` in 1.4.0), older versions of CocoaPods blow up in confusion.
+  # Specifying a `cocoapods_version >= 1.4.0` does nothing to solve the problem, as Ruby
+  # interprets unknown parameters as low-level syntax errors.
+  #
+  # Instead, we're forced to use Ruby's built-in semver comparison before declaring parameters
+  # added in recent CocoaPods releases.
+  if Version.new(Pod::VERSION) >= Version.new("1.4.0")
```

Comment:
Holy cow, that is an amazing workaround.

Reply:
It’s such a terrible hack and on many levels I wish it didn’t work. 😭